docs/csharp/whats-new/relationships-between-language-and-library.md

```diff
@@ -0,0 +1,58 @@
+---
+title: The relationship between language features and library types | Microsoft Docs 
+description: Language features often rely on library types for implementation. Understand that relationship.
+keywords: C# language design, standard library
+author: billwagner
+ms.author: wiwagn
+ms.date: 07/20/2017
+ms.topic: article
+ms.prod: .net
+ms.devlang: devlang-csharp
+---
+
+# Relationships between language features and library types
+
+The C# language definition requires a standard library to have certain
+types, and certain accessible members on those types. The compiler generates
+code that uses these required types and members for many different language
+features. When necessary, there are NuGet packages that contain types
+needed for newer versions of the language when writing code for environments
+where those types or members have not been deployed yet.
+
+This dependency on standard library functionality has been part of the
+C# language since its first version. In that version, examples included
+`System.Exception`, `System.String`, `System.Int32` and more. That first
```

Comment:
I'd make this into a bulleted list and add a brief explanation for why each type is required by the compiler.